scripts/cleanup-db.sh

```diff
@@ -71,7 +71,11 @@ docker exec "$OSRD_POSTGRES" psql -p "$OSRD_POSTGRES_PORT" -f //tmp/init.sql > /
 
 # Clear Valkey Cache
 echo "Deleting valkey cache..."
-docker exec "$OSRD_VALKEY" valkey-cli -p "$OSRD_VALKEY_PORT" FLUSHALL > /dev/null 2>&1 || docker volume rm -f osrd_valkey_data > /dev/null
+if [ "$PR_TEST" -eq 1 ]; then
+  docker exec "$OSRD_VALKEY" valkey-cli -p "$OSRD_VALKEY_PORT" FLUSHALL > /dev/null 2>&1 || docker volume rm -f osrd-pr-tests_vakey_data > /dev/null
```

Comment:
How about mutualizing more by introducing a var:
```suggestion
OSRD_VALKEY_VOLUME=osrd_valkey_data
if [ "$PR_TEST" -eq 1 ]; then
   OSRD_VALKEY_VOLUME=osrd-pr-tests_valkey_data
fi

docker exec "$OSRD_VALKEY" valkey-cli -p "$OSRD_VALKEY_PORT" FLUSHALL > /dev/null 2>&1 || docker volume rm -f "$OSRD_VALKEY_VOLUME" > /dev/null
```

(probably even possible to mutualize the compose project value for the names)

Reply:
I'm wondering if it might be nicer to have cleanup DB only provide defaults for `OSRD_VALKEY`, etc, and have the pr-test script pass those instead of `PR_TEST=1`

Reply:
Both OK to me, better if it's consistent all the way.

Reply:
> I'm wondering if it might be nicer to have cleanup DB only provide defaults for `OSRD_VALKEY`, etc, and have the pr-test script pass those instead of `PR_TEST=1`

I would agree it would be better if we were to spawn another DB on another port but I don't see the benefit of doing so as their will always be only two value possible and only one use-case. However, it could makes sense to do so if we want the user to be able to manually / separately use the cleanup script on the PR tests instance. Is it the case ?

Reply:
I'll add the volume var suggestion but with the other vars above. (I think it will make the script easier to understand if everything is defined at the same place)

Reply:
> > I'm wondering if it might be nicer to have cleanup DB only provide defaults for `OSRD_VALKEY`, etc, and have the pr-test script pass those instead of `PR_TEST=1`
> 
> I would agree it would be better if we were to spawn another DB on another port but I don't see the benefit of doing so as their will always be only two value possible and only one use-case. However, it could makes sense to do so if we want the user to be able to manually / separately use the cleanup script on the PR tests instance. Is it the case ?

I would go for a clean/simple step on the current use, and we would do another step if another use-case appears.
So probably OK to just have a clean support of `pr-tests` + regular cleanup.